main.go

```diff
@@ -1,14 +1,28 @@
 package main
 
 import (
+	"flag"
 	"fmt"
 	"log"
 	"net/http"
 	"os"
 	"os/exec"
 )
 
+var configFile = flag.String("f", "images.yaml", "retagger config (images.yaml) file to use.")
```

Comment:
I would prefer to use cobra/viper here. It would be more consistent with other projects then.

And then we might have an easier time by just switching to the structure of other commandline tools - e.g. `devctl`
https://github.com/giantswarm/devctl